pulumi/grapl/__main__.py

```diff
@@ -194,9 +190,16 @@ def main() -> None:
                         "aws_access_key_id",
                         "aws_access_key_secret",
                         "aws_endpoint",
-                        "redis_endpoint",
                     }
                 }
+                integration_test_only_job_vars = {
```

Comment:
I'll note that we probably will want to pass this into grapl core as well, but that can be done later